scheduler/src/common/constants/cgi.constant.ts

```diff
@@ -0,0 +1,18 @@
+export class CgiConstants {
+    static readonly BATCH_TYPE = 'GA'; // batch type
```

Comment:
Are these sensitive and should be in VAULT?
CC: @john-fletcher-aot @gchauhan-aot